addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -490,10 +490,11 @@ MockServer.include({
      * @returns {Object}
      */
     async _mockRouteMailThreadData(thread_model, thread_id, request_list) {
-        const res = {};
+        const res = {'hasReadAccess': true};
         const thread = this._mockSearchRead(thread_model, [[['id', '=', thread_id]]], {})[0];
         if (!thread) {
             console.warn(`mock server: reading data "${request_list}" from invalid thread "${thread_model}_${thread_id}"`);
```

Comment:
this warning should be removed then, if we now consider this a possible use case in the server, the mock server should allow it too without crashing runbot

Reply:
@seb-odoo , Done..!